CODE_OF_CONDUCT.md

```diff
@@ -10,30 +8,43 @@ We pledge to act and interact in ways that contribute to an open, welcoming, div
 
 ## Our Standards
 
-In this community we strive to go the extra step to look out for each other. Don’t just aim to be technically unimpeachable, try to be your best self. In particular, avoid flirting with offensive or sensitive issues, particularly if they’re off-topic; this all too often leads to unnecessary fights, hurt feelings, and damaged trust; worse, it can drive people away from the community entirely.
+In this community we strive to go the extra step to look out for each other and build up each others work, skills and confidence.
+Don’t just aim to be technically unimpeachable, try to be your best self.
```

Comment:
> technically unimpeachable

I know this is coming from the original CoC, but as a non-native speaker, it took me a second to even understand what that means. I'm still not 100% sure.

Is it talking about people essentially trying to get as close as possible to the limit without breaking any rules?

Reply:
```suggestion
Don’t try to push the line of what's "technically allowed", instead, try to act with maturity, kindness and grace.
```